specification/draft/apps.mdx

```diff
@@ -437,6 +439,8 @@ interface HostContext {
 
 All fields are optional. Hosts SHOULD provide relevant context. Guest UIs SHOULD handle missing fields gracefully.
 
+For `styles`, apps SHOULD provide CSS fallback values (e.g., `var(--color-text-primary, #171717)`) to handle hosts that don't supply styles.
```

Comment:
Hmm I think the best way to provide fallbacks is to just set the variable itself, and if the styles are there, it'll overwrite each of them

```
:root {
  --colorTextPrimary: #171717;
}
```